build.gradle

```diff
@@ -68,7 +69,8 @@ apply from: 'gradle/verifier.gradle'
  * Don't create `posix` and `darwin` sourceSets in single project.
  */
 def platforms = ["common", "jvm", "js", "posix", "darwin"]
-rootProject.ext.skipPublish = ["binary-compatibility-validator", "ktor-server-benchmarks", "ktor-client-benchmarks"]
+ext.skipPublish = ["binary-compatibility-validator", "ktor-server-benchmarks", "ktor-client-benchmarks"]
+ext.skipDefault = ["ktor-bom"]
```

Comment:
Unlike `skipPublish`, `skipDefault` is not self-explaining name. So for me, it is not clear what the _default_ is it and how is it skipped.

Reply:
Good catch, renamed to `nonDefaultProjectStructure`